boards/arm/lpcxpresso54114/Kconfig.defconfig

```diff
@@ -0,0 +1,43 @@
+# Kconfig - LPCXpresso54114 board
+#
+# Copyright (c) 2017, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BOARD_LPCXPRESSO54114
+
+config BOARD
+	default lpcxpresso54114
+
+if USART_MCUX_LPC
+
+config USART_MCUX_LPC_0
+	def_bool y if UART_CONSOLE
+
+endif # USART_MCUX_LPC
+
+config PINMUX
```

Comment:
make this a select in Kconfig.board

Reply:
Making it a select causes warnings:
```
warning: (BOARD_LPCXPRESSO54114) selects PINMUX which has unmet direct dependencies (BOARD_FRDM_K64F || BOARD_FRDM_KL25Z || BOARD_FRDM_KW41Z || BOARD_HEXIWEAR_K64 || BOARD_USB_KW24D512 || BOARD_GALILEO)
```

Adding `CONFIG_PINMUX=y` to lpcxpresso54114_defconfig eliminates the warnings and is more consistent with other boards.